compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -28,6 +28,22 @@
 namespace
 {
 
+bool check_shape(const luci::CircleNode *node, const std::vector<loco::Dimension> &shape)
```

Comment:
(optional) about the name... `is_shape_same` or `is_same_shape` or `is_shape_equal`

Reply:
I'll go with `is_same_shape` :)